### PR TITLE
Fixed Special Price class not added in configurable product page

### PR DESCRIPTION
**Pull request by ravi-chandra3197**
### Description (*)

Fixed Special Price class not added in configurable product page
### Fixed Issues (if relevant)

1. magento/magento2#24972: Special Price class not added in configurable product page #24972

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Swatches/view/base/web/js/swatch-renderer.js [hunk 1]
```diff
@@ -951,6 +951,11 @@ define([
             );
 
             isShow = typeof result != 'undefined' && result.oldPrice.amount !== result.finalPrice.amount;
+            if (isShow) {
+                $productPrice.find('span:first').addClass('special-price');
+            } else {
+                $productPrice.find('span:first').removeClass('special-price');
+            }
```
Comment (lbajsarowicz) [inline app/code/Magento/Swatches/view/base/web/js/swatch-renderer.js]: In my opinion you can use `toggleClass`

https://api.jquery.com/toggleclass/

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]

File: app/code/Magento/Swatches/view/base/web/js/swatch-renderer.js (hunk 1 above)
Comment (ravi-chandra3197) [reply app/code/Magento/Swatches/view/base/web/js/swatch-renderer.js]: Hello @lbajsarowicz 
I have update PR as per your suggestion
can you review it?

**Review by lbajsarowicz — APPROVED** [review]
:+1:

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6520](https://jira.corp.magento.com/browse/ENGCOM-6520) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**